lucille-core/src/main/java/com/kmwllc/lucille/indexer/SolrIndexer.java

```diff
@@ -110,6 +111,10 @@ public void closeConnection() {
 
   @Override
   protected void sendToIndex(List<Document> documents) throws Exception {
+    //when sending updates/deletes to separate collections we do not want an error sending to a particular collection to
```

Comment:
super-minor, but whitespace after the // would be in keeping with the convention